### PR TITLE
UI Support private catalog for auth

### PR DESCRIPTION
**Pull request by n313893254**
Proposed changes
======
UI Support private catalog for auth

- After: 
image

Types of changes
======
- New feature (non-breaking change which adds functionality)

Linked Issues
======
https://github.com/rancher/rancher/issues/16244
 

Further comments
======

### CODE REVIEW

**Comment by n313893254** [comment]
Backend is not merged yet

File: app/authenticated/project/project-catalogs/template.hbs [hunk 1]
```diff
@@ -14,5 +14,5 @@
 
 
 <div class="footer-actions mt-20">
-  <button {{action "goBack"}} class="btn bg-transparent">{{t 'generic.back'}}</button>
+  <button {{action "goBack"}} class="btn bg-primary">{{t 'generic.back'}}</button>
```
Comment (westlywright) [inline app/authenticated/project/project-catalogs/template.hbs]: Using the `save-cancel` as a comparison we default cancel to `bg-transparent` I think we should stick with that.

**Comment by westlywright** [comment]
@logan When the back-end is in and the changes are made can you remove the two labels that exist and add `status/ready-for-merge`?

**Comment by n313893254** [comment]
@westlywright Done. Thanks for your review.

**Comment by loganhz** [comment]
@westlywright sure

**Comment by loganhz** [comment]
Hi @westlywright backend has been merged. Can we merge this one?

**Pull request merged**